pymc3/tests/test_advi.py

```diff
@@ -83,13 +83,13 @@ def test_check_discrete_minibatch():
 
     def create_minibatch():
         while True:
-            return disaster_data
+            return disaster_data,
```

Comment:
The `,` is there to make it a tuple of length 1. 

I guess it's a poor short-hand style that I use. More explicitly, it would be `[data]` or `(data,)`